packages/vulcan-ui-material/lib/components/forms/base-controls/MuiCheckboxGroup.jsx

```diff
@@ -94,21 +95,22 @@ const MuiCheckboxGroup = createReactClass({
   },
   
   renderElement: function () {
-    const controls = this.props.options.map((checkbox, key) => {
-      let value = checkbox.value;
-      let checked = (this.props.value.indexOf(value) !== -1);
-      let disabled = checkbox.disabled || this.props.disabled;
+    const { name, options, disabled: _disabled, value:_values } = this.props.inputProperties;
```

Comment:
Why are those values imported from this.props.inputProperties instead of this.props? It seems that they were coming from inputProperties before